lib/std/fmt/format_float.zig

```diff
@@ -639,8 +639,8 @@ pub const Backend128_Tables = struct {
     const bound2 = 127;
     const adjust_q = true;
 
-    fn computePow5(i: u32) [4]u64 {
-        const base = i / FLOAT128_POW5_TABLE_SIZE;
+    fn computePow5(i: usize) [4]u64 {
```

Comment:
In my diff, but should also change the type for the other tables `Backed64_TablesSmall` and `Backend64_TablesFull`.